index.html

```diff
@@ -137,6 +143,25 @@ <h4>
           </dl>
         </section>
         <section>
+          <h4>
+            <dfn>canShareFiles()</dfn> method
+          </h4>
+          <p>
+            Each user agent should have an `allowlist` of permitted MIME types
+            and file extensions or file names. These may contain wildcards, for
+            example `*.txt` or `text/*`.
+          </p>
+          <p>
+            When the {{Navigator/canShareFiles()}} method is called with
+            argument |file_types|, run the following steps:
+          </p>
+          <ol class="algorithm">
+            <li>For each |file_type| in |file_types|, if |file_type| does not
```

Comment:
We probably want a permissions policy check here. If not allowed, throw NotAllowedError.